aim/web/middlewares/profiler/profiler.py

```diff
@@ -13,6 +13,15 @@
 from starlette.requests import Request
 from starlette.types import ASGIApp, Message, Receive, Scope, Send
 
+try:
```

Comment:
lets still keep this at class level. otherwise pyinstrument will be imported even without running with --profile flag

Reply:
then we have to re-import this on every api call

Reply:
disregard